src/Features/Core/Portable/AddImport/Remote/AbstractAddImportFeatureService_Remote.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.AddImport
     internal abstract partial class AbstractAddImportFeatureService<TSimpleNameSyntax>
     {
         private async Task<ImmutableArray<AddImportFixData>> GetFixesInRemoteProcessAsync(
-            RemoteHostClient.Session session, Document document, TextSpan span, 
+            SessionWithSolution session, Document document, TextSpan span, 
```

Comment:
In server/client model. 

Client represents a client.
Connection represents a pure/general connection between client and server.
Session represents specialized activity between client/server.

user can create/get a connection from client or user can create/get a session from client.

session will give them more convenient since it is specialized to certain activity. so most of users will just use this.

connection is basically general connection between client/server. if a user want to create new kind of session or do completely new thing on remote host, then one can get connection and build from there.